kubeflow/instance/gcp_config/nodepool-example.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is an example of your own nodepool. It's not applied by default,
+# you need to edit `./kustomization.yaml` and add `nodepool-example.yaml` in
+# resources field to include this.
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerNodePool
+metadata:
+  name: KUBEFLOW-NAME-example # {"$kpt-set":"node-pool-example"}
+# Refer to https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containernodepool
+# for detailed fields and documentation.
+spec:
+  location: "LOCATION" # {"$kpt-set":"location"}
+  # You can adjust node zones by:
+  # nodeLocations:
+  # - gcp-zone-1
+  # - gcp-zone-2
+  initialNodeCount: 2
+  autoscaling:
+    minNodeCount: 2
+    maxNodeCount: 8
+  nodeConfig:
+    machineType: n1-standard-8
+    minCpuPlatform: 'Intel Broadwell'
```

Comment:
nit: guessing no difference between `'` and `"`, pick a consistent one?

Reply:
I removed the quote, because it's also valid YAML